python/pip_install/pip_repository.bzl

```diff
@@ -339,8 +337,6 @@ py_binary(
 def _whl_library_impl(rctx):
     python_interpreter = _resolve_python_interpreter(rctx)
 
-    # pointer to parent repo so these rules rerun if the definitions in requirements.bzl change.
-    _parent_repo_label = Label("@{parent}//:requirements.bzl".format(parent = rctx.attr.repo))
```

Comment:
This ultimately did nothing since creating a label does not require the path represented by the label to exist. Plus, the `whl_library` targets should be sufficiently updated if a change is made to the root `pip_repository`